internal/uvm/scsi/backend.go

```diff
@@ -200,6 +200,7 @@ func mountRequest(controller, lun uint, path string, config *MountConfig, osType
 			MountPath:  path,
 			Controller: uint8(controller),
 			Lun:        uint8(lun),
+			Partition:  uint64(config.Partition),
```

Comment:
Need to add `Partition` as part of the `LCOWMappedVirtualDisk` request for mount and unmount

Reply:
Added.